### PR TITLE
Fix ci

### PR DESCRIPTION
**Pull request by laurentS**
This PR will appear broken at first:
- it includes the commits in !67 (which fixes problems with config files)
- The following env vars should be set in the repo's github actions:
  - `GITLAB_PRIVATE_TOKEN` to a valid one with `read_api` permissions.
  - `GITLAB_PROJECTS_TO_FETCH` optionally to test running all streams. I would suggest `meltano/sdk` along with a private token above that has full read access to the repo (including pipelines/vulnerabilities streams for which I'm getting HTTP 403)

@aaronsteers can I suggest you set these up? Once done, the CI checks on the PR should turn green, and we should be able to merge other PRs a bit more comfortably.

### CODE REVIEW

**Comment by laurentS** [comment]
Actually, it looks like you might need to start CI checks manually...

**Comment by laurentS** [comment]
Thinking a bit more about it, this could expose some sensitive information (vulnerabilities for instance). I'm not sure what the best option is:
- fiddle with the catalog to exclude the stream (this could easily be reverted by a malicious user)
- mark the stream as allowing HTTP 403 like we do for `tap-github` so that the data is not fetched, but tests pass anyway. Possibly even make that conditional on the tap running in CI by checking for some env var, or add a config option to skip sensitive streams.
- Find/create a repo that is fully open that can be used for testing

**Review by aaronsteers — APPROVED** [review]

File: tap_gitlab/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+"""Placeholder."""
```
Comment (aaronsteers) [inline tap_gitlab/__init__.py:1]:
```suggestion
"""The tap-gitlab connector."""
```

**Comment by aaronsteers** [comment]
image

Logging much after the fact...

**Pull request merged**